src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -245,67 +246,69 @@ export const SelectPage = () => {
   const SortWalletAssetsDropdown = () => <SortDropdown dropDownOptions={sortDropDownOptions} />
 
   return (
-    <Column width="full">
-      <Row
-        alignItems="flex-start"
-        position="relative"
-        paddingLeft={{ sm: '16', md: '52' }}
-        paddingRight={{ sm: '0', md: '72' }}
-        paddingTop={{ sm: '16', md: '40' }}
-      >
+    <Column
+      width="full"
+      paddingLeft={{ sm: '16', md: '52' }}
+      paddingRight={{ sm: '0', md: '72' }}
+      paddingTop={{ sm: '16', md: '40' }}
+    >
+      <Row alignItems="flex-start" position="relative">
         <FilterSidebar SortDropdown={SortWalletAssetsDropdown} />
 
         {(!isMobile || !isFiltersExpanded) && (
           // @ts-ignore
```

Comment:
is this necessary if we are removing `AnimatedBox`?

Reply:
Can be removed thanks to the newly added `any` cast of AnimatedBox. That being said its deprecation is a long term goal as part of the greater sprinkles deprecation.